### DISCUSSION TITLE
UI Editing needed

### DISCUSSION THREAD

**Discussion by Domocn** [damianeickhoff/HaCasa#35] [General]
The inability to modify the dashboard through the user interface in YAML is one of the most frustrating experiences I can imagine when creating a dashboard. As someone who doesn't have a coding background, 

I find great value in the option to edit the dashboard directly in YAML within the UI. This feature not only enhances usability but also allows for a more intuitive approach to customization. 

Unfortunately, the lack of this functionality resulted in my deletion of the theme, which is particularly disappointing because it had a stunning design and aesthetic appeal.
 
Having straightforward and accessible customization options is crucial for users like me, as it helps prevent errors and enhances the overall experience of working with the dashboard.

**Comment by damianeickhoff** [comment]
Hi @Domocn

I appreciate your feedback! I agree with you that adding HaCasa to the GUI editor mode will be easier for the overall user. Though, making it available through the GUI editor is a pretty demanding task on its own besides creating the cards.

It is definitely the plan to do that, but for now the focus is on creating the cards and making the cards as modulair as possible.

I don't think there is any reason to keep this open as a discussion because I agree and it's on the plans, but not for now.
